tools/redis.md

```diff
@@ -0,0 +1,34 @@
+---
+permalink: /redis
+layout: post
+title: Redis
+link: https://docs.redislabs.com/latest/rs/administering/product-lifecycle/#end-of-life-schedule
+activeSupportColumn: false
+command: $ redis-server --version
+releaseDateColumn: true
+releases:
+  '5.4':
+    release: 2018-12-01
+    lts: false
```

Comment:
LTS is false by default, so it none of the releases are LTS, you can drop it entirely

Reply:
Cool! I'll drop this in the next commit.